pages/1_14/items/basic_items.md

```diff
@@ -0,0 +1,113 @@
+# Basic Items
+
+At this point you should be able start with creating some basic items.
```

Comment:
Let's not presume that the reader is just reading through everything in order 😃 
Maybe state that you're presuming they've followed the previous tutorials (maybe link them?) and have a project stup and working.